datafusion/sqllogictest/test_files/array.slt

```diff
@@ -6140,21 +6140,19 @@ logical_plan
 02)--Aggregate: groupBy=[[]], aggr=[[count(Int64(1))]]
 03)----SubqueryAlias: test
 04)------SubqueryAlias: t
-05)--------Projection:
-06)----------Filter: __common_expr_3 = __common_expr_3
-07)------------Projection: substr(CAST(md5(CAST(tmp_table.value AS Utf8)) AS Utf8), Int64(1), Int64(32)) AS __common_expr_3
-08)--------------TableScan: tmp_table projection=[value]
+05)--------Projection: 
```

Comment:
There is a comment a few lines above that says:

> # TODO: this should probably be possible to completely remove the filter as always true?

We can probably update that too -- but we could do it in a follow on PR as well